src/apps/loyalty/containers/inquiries/index.tsx

```diff
@@ -22,7 +22,51 @@ const Container = styled.div`
   }
 `
 
-class Inquiries extends React.Component<RelayProps, any> {
+interface State {
+  loyalty_applicant: boolean,
+  self_reported_purchases?: string
+}
+
+type RelayMutationProps = State
+
+class UpdateCollectorProfileMutation extends Relay.Mutation<RelayMutationProps, any> {
```

Comment:
Yea, I was just starting out all in one file. At the very least I think co-locating it with this component makes the most sense, I can move to a separate file.

Lots of examples I saw online seem to have all the different mutations defined in a completely separate place, but that seems weird. I'm definitely down to keep them close to the app/components for now.